src/components/DropdownList/DropdownList.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+
+interface Props {
+  people: Person[];
+  setQuery: (query: string) => void;
+  setIsListShown: (arg: boolean) => void;
+  setSelectedPerson: (person: Person) => void;
+}
+
+export const DropdownList: React.FC<Props> = ({
+  people,
+  setQuery,
+  setIsListShown,
+  setSelectedPerson,
+}) => {
+  const handleItemClick = (person: Person) => {
+    setQuery(person.name);
+    setSelectedPerson(person);
+    setIsListShown(false);
+  };
+
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        {people.length
+          ? people.map((person) => (
+            <div
+              className="dropdown-item"
+              key={person.name}
+              onKeyDown={() => { }}
```

Comment:
Why do you need empty func in this place?

Reply:
I need a keyboard listener because of this eslint rule: "Visible, non-interactive elements with click handlers must have at least one keyboard listener". However I didn't implement any functionality when pressing the button, so I added an empty func. In short, I need it to avoid linter errors